client/packages/common/src/hooks/useQueryParams/useQueryParamsStore.tsx

```diff
@@ -23,19 +23,21 @@ export interface QueryParamsState<T extends RecordWithId> {
     filterBy: FilterBy | null;
   };
 }
+const getDirection = (isDesc: boolean): 'asc' | 'desc' =>
+  isDesc ? 'desc' : 'asc';
 
-export const { Provider: QueryParamsProvider, useStore: useQueryParamsStore } =
-  createContext<QueryParamsState<any>>();
+const queryParamsStoreContext = createContext<QueryParamsState<any>>({} as any);
```

Comment:
`{} as any`, can this be used when creating context instead of providing default or doing the optional thing that was done for `StocktakeLineErrorContext` ?

Reply:
yep - I've changed and it seems to be working fine. It's much neater, thanks